weatherAPI.py

```diff
@@ -0,0 +1,32 @@
+import requests
+from abc import ABC, abstractmethod
+
+class AbstractDataService(ABC):
+    @abstractmethod
+    def getWeatherForecast(self, latitude, longitude):
+        pass
+
+class WeatherAPIService(AbstractDataService):
+    def getWeatherForecast(self, latitude, longitude):
+        url = f"https://api.open-meteo.com/v1/forecast?latitude=41.85&longitude=-87.65&hourly=temperature_2m"
+        response = requests.get(url)
+        if response.status_code == 200:
+            return response.json()
+        else:
+            raise Exception("Failed to fetch weather forecast")
```

Comment:
few things here
- the url contains hard-coded request parameters instead of utilizing the passed in `latitude` and `longitude`
- the response is not parsed down enough (you simply return `response.json()`, instead find the specific fields in the response you need, and only return the values you find there (maybe with some validation)